guids-generator.html

```diff
@@ -77,8 +79,43 @@
         listItem: {
             padding: "0",
         },
+        table: {
+            padding: "0",
+        },
+        copyButton: {
+            float: "right",
+        },
+        copyButtonSuccess: {
```

Comment:
Can we ensure both buttons have the same size? The jumping effect after copying when the label changes is not great.

Additionally, lets make all button labels ("Generate GUIDs" included) normalcase instead of uppercase.